opentracing-jdbc/opentracing-jdbc/src/test/java/org/zalando/opentracing/jdbc/DataSourceTracerTest.java

```diff
@@ -0,0 +1,236 @@
+package org.zalando.opentracing.jdbc;
+
+import io.opentracing.mock.MockSpan;
+import io.opentracing.mock.MockTracer;
+import org.h2.jdbcx.JdbcDataSource;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.zalando.opentracing.jdbc.operation.StoredProcedureOperationName;
+import org.zalando.opentracing.jdbc.span.StaticSpanDecorator;
+
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Collection;
+import java.util.List;
+
+import static java.util.Collections.singletonMap;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasEntry;
+import static org.hamcrest.Matchers.hasKey;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class DataSourceTracerTest {
+
+    private final MockTracer tracer = new MockTracer();
+    private final DataSourceTracer unit = new DataSourceTracer(tracer)
+            .withAdditionalSpanDecorators(new StaticSpanDecorator(singletonMap("flow_id", "REcCvlqMSReeo7adheiYFA")));
+    private final JdbcDataSource original = new JdbcDataSource();
+
+    DataSourceTracerTest() {
+        original.setUrl("jdbc:h2:mem:users;DB_CLOSE_DELAY=-1");
+    }
+
+    @BeforeEach
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/whiskeysierra/tracer/pullRequest?prid=4563925)